<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DicomBrowserWin</class>
 <widget class="QMainWindow" name="DicomBrowserWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dicom Browser (RESEARCH ONLY)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/icon.png</normaloff>:/icons/icon.png</iconset>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="seriesSplit">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="sourceGroup">
       <property name="title">
        <string>Series</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="importDirButton">
            <property name="toolTip">
             <string>Import a directory containing DICOM files</string>
            </property>
            <property name="text">
             <string>Import Dir...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="importZipButton">
            <property name="toolTip">
             <string>Import a zip file containing DICOM files</string>
            </property>
            <property name="text">
             <string>Import Zip...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="statusText">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="statusProgressBar">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Import progress</string>
            </property>
            <property name="maximum">
             <number>1</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="format">
             <string>%p% (%v / %m)</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QTreeView" name="seriesView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="viewMetaSplitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="handleWidth">
        <number>10</number>
       </property>
       <widget class="QGroupBox" name="view2DGroup">
        <property name="title">
         <string>2D View</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="metadataGroup">
        <property name="title">
         <string>Metadata</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="0">
          <widget class="QTreeView" name="attrView">
           <property name="toolTip">
            <string>List of DICOM tags</string>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="textElideMode">
            <enum>Qt::ElideNone</enum>
           </property>
           <property name="indentation">
            <number>10</number>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Filter:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filterLine">
             <property name="toolTip">
              <string>Filter DICOM tags by name, ID, or content, using regular expressions, </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="numLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="imageSlider">
        <property name="toolTip">
         <string>Position in selected series, sorted in alphabetical filename order</string>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoLevelsCheck">
        <property name="toolTip">
         <string>Automatically scale the value range in the image view, uncheck to use current levels across all images in series</string>
        </property>
        <property name="text">
         <string>Auto Levels</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Open_Directory">
   <property name="text">
    <string>&amp;Open Directory</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
